### PR TITLE
refactor(console): tonal palette colors and theme mixins

### PR DESCRIPTION
**Pull request by demonzoo**
## Summary

Refactor tonal palette colors in admin console, and prepare mixins for dark theme.

## Testing

-[x] Confirmed by UX designer

### CODE REVIEW

File: packages/console/src/scss/_colors.scss [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+@mixin dark-theme {
+  --color-surface: var(--color-neutral-10);
+  --color-on-surface: var(--color-neutral-80);
+  --color-on-surface-variant: var(--color-neutral-variant-80);
+  --color-primary: var(--color-primary-80);
+  --color-on-primary: var(--color-primary-20);
+  --color-error: var(--color-error-80);
+  --color-on-secondary-container: var(--color-secondary-90);
+  // to-do: add more dark theme colors
+}
```
Comment (gao-sun) [inline packages/console/src/scss/_colors.scss:91]: will it be better to remove this until we really have a dark theme?

Comment (demonzoo) [reply packages/console/src/scss/_colors.scss:91]: I think it's OK to keep this, as these colors were also grabbed from our UX design (from dark theme section). They might be slightly updated though, but with these foundamentals, we are `dark theme ready` on code level. Just need to add more colors to the mixin when we are 100% ready on design level.

Comment (demonzoo) [reply packages/console/src/scss/_colors.scss:92]: Added comment `Currently not in use` in [AppContent/index.module.scss](https://github.com/logto-io/logto/pull/369/files#diff-044c75c192502b16571271b330b2db1b1b10d9e2dd5f38ee0193634d0a91ab79)

**Review by xiaoyijun — APPROVED** [review]
LGTM

**Pull request merged**